src/client.ts

```diff
@@ -464,10 +462,10 @@ function changeDocument() {
   let ide = client.ide;
   // @FIXME: This is not right in the non-internal case.
   let docId = "/examples/quickstart.eve";
-  let path = "/" + location.hash.split('?')[0].split("#/")[1];
-  if(path) {
+  let path = location.hash && location.hash.split('?')[0].split("#/")[1];
+  if(path && path.length > 2) {
```

Comment:
In order to be remotely valid it needs to contain at least a character and a slash. Honestly it should be 3, since all files must at least be <workspace>/<file>